src/containers/tutor-home-page/language-step/LanguageStep.jsx

```diff
@@ -1,17 +1,48 @@
-import Box from '@mui/material/Box'
+import { useState } from 'react'
 
+import Box from '@mui/material/Box'
 import { styles } from '~/containers/tutor-home-page/language-step/LanguageStep.styles'
+import AppSelect from '~/components/app-select/AppSelect'
 import img from '~/assets/img/tutor-home-page/become-tutor/languages.svg'
+import { languagesMock } from '../subjects-step/constants'
 
 const LanguageStep = ({ btnsBox }) => {
+  const [language, setLanguage] = useState('')
+
+  // Подготовка полей для AppSelect
+  const languageFields = languagesMock.map((language) => ({
+    title: language.name,
+    value: language.name
+  }))
+
   return (
     <Box sx={styles.container}>
       <Box sx={styles.imgContainer}>
         <Box component='img' src={img} sx={styles.img} />
       </Box>
       <Box sx={styles.rigthBox}>
-        <Box>Language here</Box>
-        {btnsBox}
+        <Box sx={styles.contentBox}>
+          <Box sx={styles.textBox}>
+            Please select the language in which you would like to study and
```

Comment:
May be use translation here? and update it if needed...

Reply:
I agree with comment, use translation for all labels and titles.

Reply:
Fix this bug)